samples/shopping-cart-quickstart/kubernetes/deployment.yml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1
```

Comment:
These kubernetes yml are for my testing right now. Will be moved to separate sample and documentation.

Reply:
and yes, it works in local minikube with local postgres